portality/lib/val_convert.py

```diff
@@ -0,0 +1,43 @@
+"""
+contain functions or factory which provide value conversion(coerce) in
+SeamlessMixin or DataObj layer
+"""
+
+from typing import Callable, Any
+
+
+def create_fn_to_isolang(output_format=None, is_upper=False) -> Callable[[Any], str]:
```

Comment:
I'm not sure why this has got its own file, can't this just live in the `portality.lib.coerce` module?

Reply:
`create_fn_to_isolang` also call by `portality.lib.dataobj`
I should not let `portality.lib.coerce` import to `portality.lib.dataobj` or either way.

you can find a lot of same kind of convert function in `portality.lib.coerce` or `portality.lib.dataobj`, some of those is duplicate 
* seamless.to_utf8_unicode
* seamless.to_unicode_upper
* to_country_code
* to_url
* to_float

in order to solve the import and duplicate problems, extract to new file to contain list of same kind of function is a solution.

but, purpose of this task is not refactoring. that is the reason why this file only have one function.